options/locale/locale_en-US.ini

```diff
@@ -1060,58 +1060,58 @@ release.stable = Stable
 release.edit = edit
 release.ahead = <strong>%d</strong> commits to %s since this release
 release.source_code = Source Code
-release.new_subheader = Publish releases to keep track of project versions.
-release.edit_subheader = A detailed changelog can help users understand what has been changed.
+release.new_subheader = Releases organize project versions.
+release.edit_subheader = Releases organize project versions.
 release.tag_name = Tag name
 release.target = Target
-release.tag_helper = Choose an existing tag, or create a new tag.
+release.tag_helper = Choose an existing tag or create a new tag.
 release.title = Title
 release.content = Content
 release.write = Write
 release.preview = Preview
 release.loading = Loading…
-release.prerelease_desc = This is a pre-release
-release.prerelease_helper = We'll point out that this release is not production-ready.
+release.prerelease_desc = Mark as Pre-Release
+release.prerelease_helper = Mark this release unsuitable for production use.
 release.cancel = Cancel
 release.publish = Publish Release
 release.save_draft = Save Draft
-release.edit_release = Edit Release
-release.delete_release = Delete This Release
-release.deletion = Release Deletion
-release.deletion_desc = Deleting this release will delete the corresponding Git tag. You will not lose any code. Do you want to continue?
+release.edit_release = Update Release
+release.delete_release = Delete Release
+release.deletion = Delete Release
+release.deletion_desc = Deleting a release removes its Git tag from the repository. Repository contents and history remain unchanged. Continue?
 release.deletion_success = The release has been deleted.
-release.tag_name_already_exist = Release with this tag name already exists.
-release.tag_name_invalid = Tag name is not valid.
+release.tag_name_already_exist = A release with this tag name already exists.
+release.tag_name_invalid = The tag name is not valid.
 release.downloads = Downloads
 
-branch.name = Branch name
+branch.name = Branch Name
 branch.search = Search branches
-branch.already_exists = A branch named %s already exists.
+branch.already_exists = A branch named '%s' already exists.
 branch.delete_head = Delete
-branch.delete = Delete Branch %s
+branch.delete = Delete Branch '%s'
 branch.delete_html = Delete Branch
-branch.delete_desc = Deleting a branch is permanent. There is no way to undo it.
-branch.delete_notices_1 = - This operation <strong>CANNOT</strong> be undone.
-branch.delete_notices_2 = - This operation will permanently delete everything in branch %s.
-branch.delete_notices_html = - This operation will permanently delete everything in branch
-branch.deletion_success = %s has been deleted.
-branch.deletion_failed = Failed to delete branch %s.
-branch.delete_branch_has_new_commits = %s cannot be deleted because new commits have been added after merging.
+branch.delete_desc = Deleting a branch is permanent. It <strong>CANNOT</strong> be undone. Continue?
+branch.delete_notices_1 = 
```

Comment:
These lines should be probably removed here and also from template files if not needed anymore

Reply:
I'll see what I can do. I don't want to shuffle around too much stuff, but I understand that you don't want empty strings lying around.

Reply:
I've removed four references to `branch.delete_notices_*` strings, I think that's all.